std_msgs/String missionName

---

std_msgs/Bool result
